### DISCUSSION TITLE
Does Prisma2 Support Postgres Ltree

### DISCUSSION THREAD

**Discussion by aakashkag** [prisma/prisma#2494] [Q&A] [answered]
Hi
I am dealing with Tree structure data. I found that Ltree is solution in Postgres. 
Does Prisma2 Support Ltree?
Is there any example?
If Prisma doesn't support Ltree then any future plan to support?

Thank You :)

**Comment by tomhoule** [comment, marked as answer]
Unfortunately it doesn't at the moment, but we absolutely want to support more native database types in the future. Please open an issue if ltree support matters to you, so progress on this can be tracked :)

One workaround for now would be using the `raw` method, falling back to raw SQL.

**Comment by ludralph** [comment]
Hi @aakashkag,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
